src/wazuh_db/schema_agents.sql

```diff
@@ -375,6 +375,10 @@ CREATE TABLE IF NOT EXISTS vuln_cves (
     version TEXT,
     architecture TEXT,
     cve TEXT,
+    detection_time TEXT DEFAULT '',
```

Comment:
Not sure if this field should be `TEXT` or `INTEGER` (as a timestamp)

Reply:
As we agreed we are going to let this field as TEXT, since it is how we have been working on.